Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathFinder.php

```diff
@@ -257,9 +257,63 @@ private function fetchSingle(string $where, array $parameters): DocumentNodeInfo
         return DocumentNodeInfo::fromDatabaseRow($row);
     }
 
+    /**
+     * @param string $where
+     * @param array<string> $parameters
+     * @return DocumentNodeInfo[]
+     */
+    private function fetchMultiple(string $where, array $parameters): array
+    {
+        try {
+            $rows = $this->dbal->fetchAllAssociative(
+                'SELECT * FROM ' . $this->tableNamePrefix . '_uri
+                     WHERE ' . $where,
+                $parameters,
+                DocumentUriPathProjection::COLUMN_TYPES_DOCUMENT_URIS
+            );
+        } catch (DBALException $e) {
+            throw new \RuntimeException(sprintf(
+                'Failed to load node for query "%s": %s',
+                $where,
+                $e->getMessage()
+            ), 1683808640, $e);
+        }
+
+        return array_map(fn($row) => DocumentNodeInfo::fromDatabaseRow($row), $rows);
```

Comment:
can be simplified to:
```suggestion
        return array_map(DocumentNodeInfo::fromDatabaseRow(...), $rows);
```